main.go

```diff
@@ -48,9 +48,6 @@ import (
 	sriovnetworkv1 "github.com/k8snetworkplumbingwg/sriov-network-operator/api/v1"
 	"github.com/k8snetworkplumbingwg/sriov-network-operator/controllers"
 	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/featuregate"
-	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/leaderelection"
```

Comment:
I think we can delete this package completely

And this will increase our code coverage :rocket:

Reply:
nice catch thanks!